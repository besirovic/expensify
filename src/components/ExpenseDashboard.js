import React from 'react';

import ExpenseList from './ExpenseList';
import ExpensesSummary from './ExpensessSummary';
import ExpenseListFilters from './ExpenseListFilters';

const ExpenseDashboard = (props) => {
    return (
        <div>
            <ExpensesSummary />
            <hr />
            <ExpenseListFilters />
            <hr />
            <ExpenseList />
        </div>
    );
};

export default ExpenseDashboard;