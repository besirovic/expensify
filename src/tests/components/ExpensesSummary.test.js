import React from 'react';
import { shallow } from 'enzyme';

import { ExpensesSummary } from '../../components/ExpensessSummary';

test('should render correct message for 1 expense', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={1} expensesTotal={235} />);
    expect(wrapper).toMatchSnapshot();
});

test('should render correct message for multiple expense', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={23} expensesTotal={23512340987} />);
    expect(wrapper).toMatchSnapshot();
});