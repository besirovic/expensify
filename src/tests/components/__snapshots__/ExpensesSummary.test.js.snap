// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correct message for 1 expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totalling 
    $2.35
  </h1>
</div>
`;

exports[`should render correct message for multiple expense 1`] = `
<div>
  <h1>
    Viewing 
    23
     
    expenses
     totalling 
    $235,123,409.87
  </h1>
</div>
`;
